res/skins/Deere/sampler_controls_row.xml

```diff
@@ -77,19 +77,6 @@
         <Layout>horizontal</Layout>
         <ObjectName>ButtonGrid</ObjectName>
         <Children>
-
-          <Template src="skin:left_display_2state_button.xml">
```

Comment:
I plan to work a bit more on the samplers layout, but this must not be done in this PR

Reply:
@esbrandt Will you take care of this before 2.1 release or should I?
https://bugs.launchpad.net/mixxx/+bug/1743109 is almost fixed, except Deere samplers are lacking hotcues. Hotcues 5-8 would need to be in a second hotcue row on small screens (this can be constructed responsive, like some things in Tango)